pages/sign-in.tsx

```diff
@@ -181,7 +187,21 @@ function SignInPage(props: any) {
           }}
         />
 
+        <div className={styles.actions} style={{ marginTop: 10 }}>
+          <input
+            type="checkbox"
+            onClick={() => {
+              if (state.adminLogin === 'false') {
```

Comment:
why not make `adminLogin` bool and just do a one-liner `setState({ ...state, adminLogin:  !adminLogin})`

Reply:
Modified on the latest commit.